packages/fuse2.rb

```diff
@@ -0,0 +1,26 @@
+require 'package'
+
+class Fuse2 < Package
```

Comment:
Does this package have any files that conflict with the other fuse package?

Reply:
it doesnt conflict at all, I think they're naturally supposed to be that way.

Reply:
no.